index.js

```diff
@@ -5,32 +5,16 @@
 process.chdir(__dirname);
 
 // Perform node version check before loading any other files or modules
-// Doing this check as soon as possible allows us to avoid ES6 parser errors or
-// other issues
-// Try to display messages nicely, but gracefully degrade if anything goes wrong
+// Doing this check as soon as possible allows us to
+// avoid ES6 parser errors or other issues
 const pkg = require("./package.json");
 
 if (!require("semver").satisfies(process.version, pkg.engines.node)) {
-	let colors;
-	let log;
-
-	try {
-		colors = require("colors/safe");
```

Comment:
We switched `colors` to `chalk`. We can't really use it here though because `chalk` targets Node 6+, and that makes it useless here.